Blockzilla/SettingsViewController.swift

```diff
@@ -280,13 +276,15 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
         tableView.allowsSelection = true
         tableView.estimatedRowHeight = 44
 
-        toggles = initialToggles
-        for (i, blockerToggle) in toggles {
-            let toggle = blockerToggle.toggle
-            toggle.onTintColor = UIConstants.colors.toggleOn
-            toggle.addTarget(self, action: #selector(toggleSwitched(_:)), for: .valueChanged)
-            toggle.isOn = Settings.getToggle(blockerToggle.setting)
-            toggles[i] = blockerToggle
+        initializeToggles()
+        for (i,toggleArray) in toggles{
+            for (j, blockerToggle) in toggleArray{
```

Comment:
It's not n^2 because the n toggles are spread amongst the i sections, each with a variable j number of blockerToggles. I got rid of nils, so every blockerToggle will be an actual blockerToggle. This is O(n).